include/deal.II/base/numbers.h

```diff
@@ -81,13 +81,13 @@ namespace internal
      * Maximal vector length of VectorizedArray for double.
      */
     constexpr static unsigned int max_width =
-#if DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 1 && defined(__ALTIVEC__)
+#if DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 128 && defined(__ALTIVEC__)
```

Comment:
why is altivec a special case here? Can't we just use the width and sort all entries here?